firmware/arduino/chessboard/motorDriver.ino

```diff
@@ -41,22 +41,42 @@ void moveDiagonal(int dirX, int dirY, int spacesX, int spacesY)
 
   // similary subject to change as above function dictation, same number as above though
   float numStepsX = spacesX * stepsPerSpace;
-  float numStepsX = spacesY * stepsPerSpace;
+  float numStepsY = spacesY * stepsPerSpace;
 
   digitalWrite(MOTOR_SLEEP, HIGH);
   digitalWrite(MOTOR_RESET, HIGH);
   digitalWrite(MOTOR_ENABLE, LOW);
 
   digitalWrite(xMotor[1], dirX);
-  digitalWrite(yMotor[1], dirX);
-
-  for(i = 0; i < numSteps; i++)
+  digitalWrite(yMotor[1], dirY);
+  
+  if (numStepsX = numStepsY) 
   {
+    for(i = 0; i < numStepsX; i++)
+    {
     digitalWrite(xMotor[0], LOW);
```

Comment:
The lines in the loop should be indented by one

Reply:
Apparently VS code has [auto formatting](https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode), hopefully this is useful for your future PRs, you can just auto format before pushing the code!